src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -232,10 +232,13 @@ def geoprocess(endpoint, data, retry=None):
     try:
         response = requests.post(geop_url,
                                  data=json.dumps(data),
-                                 headers={'Content-Type': 'application/json'})
+                                 headers={'Content-Type': 'application/json'},
+                                 timeout=200)
```

Comment:
I tried to find where we set the Celery hard timeout to 300, but couldn't find it at a quick glance.  Is it possible to set this value as a function of that setting since the relationship is important?

Reply:
Yes, we don't set it currently; it has a default of `300`. I will set it explicitly, and set a request timeout setting based on it.